pkg/controller/sparkapplication/sparkapp_util.go

```diff
@@ -157,6 +161,18 @@ func getContainerTerminatedState(name string, podStatus apiv1.PodStatus) *apiv1.
 	return nil
 }
 
+func getContainerRunningState(name string, podStatus apiv1.PodStatus) *apiv1.ContainerStateRunning {
```

Comment:
Looks like you can name this method `isDriverContainerRunning`.

Reply:
Thanks, renamed this method as `isDriverContainerRunning`